src/encoded/schemas/experiment_mic.json

```diff
@@ -118,6 +118,28 @@
                 "title": "Image ID",
                 "type": "string"
             }
+        },
+        "extinguishing_method": {
+            "title" : "Fluorescence Extinguishing Method",
+            "description": "Method of extinguishing fluorescent signal after an imaging cycle",
+            "type" : "string",
+            "lookup": 68,
+            "suggested_enum": [
+                "DNase treatment",
+                "fluorophore cleavage",
+                "photobleaching",
+                "strand displacement"
+            ]
+        },
+        "fiducial_label": {
+            "title": "Fiducial Label",
+            "description": "Type of fiducial label used for drift correction during sequential imaging",
+            "type": "string",
+            "lookup": 69,
```

Comment:
Not that we've actually implemented it in a rigorous way but if these fields are specific to sequential imaging techniques as indicated by the descriptions would it make sense to add a "use_for": "sequential imaging" similar to how we do for some fields in for example ExperimentSeq?

Reply:
Sure, I didn't know about this. How is it used? By submit4dn or FFedit-create?